Library/Formula/j2objc.rb

```diff
@@ -0,0 +1,40 @@
+class J2objc < Formula
+  homepage "http://j2objc.org"
+  url "https://github.com/google/j2objc/archive/0.9.6.1.tar.gz"
+  sha256 "ff3e1da035dbaa4fc19e59cb74e3420b252bc328abb3b0e46acfce95416fdb3a"
+  head "https://github.com/google/j2objc.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.7+"
+
+  def install
+
+    # Call make dist to build the full distribution
+    system "make", "dist", "-j4"
```

Comment:
Don't hardcode the job count - Homebrew sets `MAKEFLAGS` with the appropriate `-j` already.